src/main/resources/view/HelpWindow.css

```diff
@@ -3,17 +3,48 @@
 }
 
 #copyButton {
-    -fx-background-color: dimgray;
+    -fx-background-color: #3C4EEF;
```

Comment:
I like how you adapted our UI to look more like the design Jingya made:)

Reply:
Yeah, thanks for doing this!